components/crud-web-apps/volumes/README.md

```diff
@@ -0,0 +1,42 @@
+# Jupyter web app
```

Comment:
Should be changed to `Volumes Web App`.

Reply:
Yeap, copy paste bit me here

Reply:
pushed a commit for this, thanks for catching it!